README.md

```diff
@@ -15,33 +15,36 @@
 
 All of these tools must be run on the same host as ClickHouse server is running.
 
-## Local development (using venv)
+## Local development (using poetry)
 
 ```sh
-python3 -m venv venv
-source venv/bin/activate
-pip install .[test]
-flit build --no-use-vcs
-flit install
+sudo make install-poetry
+# or to install in user's home directory
+make POETRY_HOME=~/opt/poetry install-poetry
+
+# For building deb packages
+sudo make prepare-build-deb
+sudo make build-deb-package  
 
 # lint
 make lint
 
 # unit tests
-pytest
+make unit-tests
+make unit-tests PYTEST_ARGS="-k test_name"
 
 # integration tests (rebuild docker images using a .whl file)
-cd tests; behave
+make integration-tests
 
 # integration tests (do not rebuild docker images)
 # useful when you didn't change source code
-cd tests; behave -D skip_setup
+make integration-tests BEHAVE_ARGS="-D skip_setup"
```

Comment:
It would be good to unify naming of targets in `ch-tools` and `ch-backup`.

In ch-backup targets are named `test-integration` and `test-unit`.
https://github.com/yandex/ch-backup/blob/8737ac84fe13f82ddb7be37e97b5b9a89fc8e18d/Makefile#L58

I have no preference on naming schema. We can rename targets either here or in ch-backup.

Also it makes sense to unify semantic of targets. In ch-backup the target `test-integration` implies `"-D skip_setup"`. And there is a logic in makefile that avoid docker image rebuild if there are no source code changes.

Reply:
Agree. I changed test target names like in ch-backup and added implicit `-D skip_setup` . As for other changes (manipulation of inetgration testing environments) I'd propose to to add it in a separate PR.